Stratis.Bitcoin/FullNode.cs

```diff
@@ -96,6 +73,11 @@ public class FullNode : IFullNode
         /// <summary>Application life cycle control - triggers when application shuts down.</summary>
         private NodeLifetime nodeLifetime;
 
+        public interface IConsoleLogger
```

Comment:
Good attempt but I already had a different idea in mind, create a new separate Interface 
```
IFeatureStats // or IFeatureBench
{
     string GetState()  // or AddStats(StringBuilder stringBuilder)
}
```
Then when calling it
```
 foreach (var feature in this.Services.Features.OfType<IFeatureStats>())
       ...
```

WDYT